### ISSUE TITLE
graphql shows : "no schema is available"

### ISSUE THREAD

**Issue by Gabriel-Andy** [gatsbyjs/gatsby#27253]
hello everyone.
it is my first time opening an issue on github.
I am building my portfolio website with gatsby.js and strapi.
I set all my contentTypes and data on on strapi.
I open two terminals, one for strapi and one for gatsby.
to test everything, I ran the command "npm run develop " on strapi and I have a status 200.
this is the response https://www.screencast.com/t/YcIiZv9n
also, I made a request by typing http://localhost:1337/jobs and I got my data back
this is the link that point to my response https://www.screencast.com/t/lwFXPjY3w.
I opened another terminal to run "gatsby develop"
this is the response I get back https://www.screencast.com/t/hsjVmM0CYFb
now when I try to access http://localhost:8000/___graphql
what I got returned is that "no schema available ".
this is the link that point to it https://www.screencast.com/t/0FYKHxgn
when I console.log to figured out what is happening
this is what I got on my console.log https://www.screencast.com/t/Zo0MSeEnDU

Gatsby CLI version: 2.12.101
Gatsby version: 2.24.7

"strapi": "3.1.6",
"strapi-admin": "3.1.6",
in summary, I can access my resource on strapi but when I request the same data using graphql like this
http://localhost:8000/___graphql nothing comes back. I basically have as returned "no schema is available
this is the gatsby repository https://github.com/Gabriel-Andy/my-portfolio-site
this is the strapi repository https://github.com/Gabriel-Andy/portfolio-api
I look forward to hearing from you guys back
thanks

**Comment by LekoArts** [comment]
Thank you for opening this!

This sounds like https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/issues/108 and when I check your `package-lock.json` file I see that you're on version `2.24.7`. This is a 2 month old version and upgrading to the latest version should fix your issue.

We're marking this issue as answered and closing it for now but please feel free to comment here if you would like to continue this discussion. We also recommend heading over to our [communities](https://www.gatsbyjs.org/contributing/community/#where-to-get-support) if you have questions that are not bug reports or feature requests. We hope we managed to help and thank you for using Gatsby!
